src/main/scala/com/databricks/labs/overwatch/utils/Config.scala

```diff
@@ -388,8 +397,73 @@ class Config() {
     }
   }
 
+  /**
+   * Function derives the cloud from workspaceUrl.
+   * @return
+   */
+  private def deriveCloudProvider(): String = {
+    if (_workspaceUrl.toLowerCase().contains("azure")) "azure"
+    else if (_workspaceUrl.toLowerCase().contains("aws")) "aws"
+    else "gcp"
+  }
```

Comment:
else should not be "gcp". I added a commit to switch this to a case statement and fail if something other than supported cloud is identified. I think my new commit will break PVC though -- we need to look more closely at the isPVC rule and make sure we handle this -- all PVC will be AWS at this time.

Reply:
changed the code to handle PVC